frontend.py

```diff
@@ -49,11 +55,14 @@ def my_form():
 @cross_origin(origin='*')
 def my_form_post():
     username = request.form['text']
+    if is_real_name(usermame):
```

Comment:
`usermame`? Does it even work? 😅 Did you test it?

Reply:
Added this in the end 😅.

Reply:
Done